monkey_island/cc/resources/telemetry.py

```diff
@@ -70,6 +71,11 @@ def telemetry_to_displayed_telemetry(telemetry):
                 monkey_label = telem_monkey_guid
             x["monkey"] = monkey_label
             objects.append(x)
+            if x['telem_type'] == 'system_info_collection' and 'credentials' in x['data']:
+                for user in x['data']['credentials']:
+                    if -1 != user.find(','):
+                        new_user = user.replace(',', '.')
```

Comment:
I think without the if we might get stuck in an infinite loop, and even if not, it feels just inefficient.
the loop goes through all the user dictionaries in x['data']['credentials'], and if there's a '.', in the username, it replaces it with a ','. The replacement works by removing the user from the dictionary and re-adding it (because the username is the key). I'm unsure if re-adding the user will make the loop go through it, and if it does, we'll be stuck in an infinite loop.